compiler/record-hessian/src/HessianComputer.cpp

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "record-hessian/HessianComputer.h"
+
+#include <luci/IR/CircleQuantParam.h>
+
+namespace record_hessian
+{
+
+/**
+ * @brief unfold the vector with NHWC shape, inherently acting in an in-place manner.
+ * @note (N, H, W, C) -> (N, L, H*W*C).
```

Comment:
Perhaps the comment has to be updated?

```suggestion
 * @note (N, H, W, C) -> (N, L, K_h * K_w * C).
```